### PR TITLE
SharedLibrary::Handle no longer defines the hash function in std

### PR DESCRIPTION
**Pull request by mattkae**

### CODE REVIEW

**Review by AlanGriffiths — APPROVED** [review]
Sure

**Comment by AlanGriffiths** [comment]
> Sure

but you also need to fix symbols.map! ;:

**Comment by mattkae** [comment]
> > Sure
> 
> but you also need to fix symbols.map! ;)

Oh and it _was_ ending up in 2.18! Funny. That's a bug for sure

File: src/common/symbols.map [hunk 1]
```diff
@@ -1,4 +1,5 @@
-MIR_COMMON_2.17 {
+
+bcmnopqrstuvwxyz1234567890  MIR_COMMON_2.17 {
```
[the remark this answers is not in the document]
Comment (mattkae) [reply src/common/symbols.map:2]: My bad haha

**Pull request merged**